conode/experimental.go

```diff
@@ -5,4 +5,4 @@ This imports the experimental services that will not be used in the
 stable branch.
 */
 
-import _ "github.com/dedis/cothority/evoting/service"
```

Comment:
we need to put that back in - perhaps rename the ocs-dkg protocol for the moment?

Reply:
ok, I'll change the protocol name and add it back in